examples/erc1155-metadata-uri/tests/abi/mod.rs

```diff
@@ -0,0 +1,17 @@
+#![allow(dead_code)]
+use alloy::sol;
+
+sol!(
+    #[sol(rpc)]
+    contract Erc1155 {
```

Comment:
This subset of functions is sufficient for these tests, but I noticed that [some other contract examples](https://github.com/OpenZeppelin/rust-contracts-stylus/blob/main/examples/erc721-metadata/tests/abi/mod.rs) have the full set even though it's not necessary.